CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+## Contributing
+
+We welcome contributions to the FireFly Project in many forms, and
+there's always plenty to do!
+
+Please visit the
+[contributors guide](https://hyperledger.github.io/firefly//contributors/contributors.html) in the
```

Comment:
Probably an issue with the core repo too, but this page doesn't appear to exist.

Reply:
Changing to https://hyperledger.github.io/firefly/contributors/